package-res/ccc/core/base/chart/chart.js

```diff
@@ -217,6 +240,33 @@ def
             isRootInit = isRoot && !isMultiChartOverflowRetry && !this.data,
             hasMultiRole;
 
+        //CDF603
+        /* Save plots layout information if the preserveLayout option is specified as true 
+           This has to be done before cleanup */
+        if(this.options.preserveLayout && this.plotPanelList && this.plotPanelList.length){
+
+            this.preservedPlotsLayoutInfo = {};
+            this.preservedPlotsLayoutInfoList = [];
+
+            this.plotPanelList.forEach(function(plotPanel) {
+                var id = plotPanel.plot.id;
+
+                this.preservedPlotsLayoutInfo[id] = {
+                    margins     : plotPanel.getLayoutMargins(),
```

Comment:
You can align values, but colons are always placed right next to property names.